include/alpaka/workdiv/WorkDivMembers.hpp

```diff
@@ -83,6 +97,14 @@ namespace alpaka
         Vec<TDim, TIdx> m_threadElemExtent;
     };
 
+    //! Deduction guide for the constructor which can be called without explicit template type parameters
+    ALPAKA_NO_HOST_ACC_WARNING
+    template<typename TDim, typename TIdx>
+    ALPAKA_FN_HOST_ACC explicit WorkDivMembers(
+        alpaka::Vec<TDim, TIdx> const& gridBlockExtent,
+        alpaka::Vec<TDim, TIdx> const& blockThreadExtent,
+        alpaka::Vec<TDim, TIdx> const& elemExtent) -> WorkDivMembers<TDim, TIdx>;
+
```

Comment:
```suggestion
    template<typename TDim, typename TIdx>
    ALPAKA_FN_HOST_ACC WorkDivMembers(
        alpaka::Vec<TDim, TIdx> const& gridBlockExtent,
        alpaka::Vec<TDim, TIdx> const& blockThreadExtent,
        alpaka::Vec<TDim, TIdx> const& elemExtent) -> WorkDivMembers<TDim, TIdx>;
```
I am not even sure if doxygen supports documenting deduction guides. I would just leave it here.

Different question: I did not need a deduction guide for g++-13. Is the deduction guide provided as a workaround for some compilers? If yes, could you maybe document them here, so we can remove the deduction guide when we phase out those compilers?

Reply:
There were many compilers, yes I can document. Thanks. ( I might return to create... functions idea because now I am trying to call WorkDivMembers constructor without constraining the arguments to alpaka vector.  )

Reply:
The comment is important for the developers